src/pages/node-express/lesson4-m1.mdx

```diff
@@ -0,0 +1,25 @@
+---
+layout: "../../layouts/genericMarkdownFile.astro"
+title: "Node Express Lesson 4: Middleware, REST Methods, and Postman"
+description: Material link for lesson 4 of the node/express class
+---
+
+import {
+  AssignmentLinkButton,
+  NextLinkButton,
+  PrevLinkButton,
+} from "../../components/navButtons.js";
+
+# Node Express Lesson 4
+
+# Middleware, REST Methods, and Postman
+
+You will finish watching **[the video from the last few weeks](https://youtu.be/Oe421EPjeBE?t=13246)**. This week, you watch from 6:10:46 of the video to the end. The initial focus is on middleware. Then each of the HTTP methods involved in API development are covered: `GET`, `POST`, `PUT`, `PATCH`, and `DELETE`. As routes for each of these API methods are implemented, you will test them using Postman. You then refactor the code, to separate out the router and controller functions.
```

Comment:
The old text said:

> You continue watching \*\*\[this video\\\](https://youtu.be/Oe421EPjeBE?t=13246)\*\*. This week, you watch from 3:40:48 of the video to 6:10:46.

but that's what was done in **Lesson 3**. https://github.com/Code-the-Dream-School/school-pages/blob/staging/src/pages/node-express/lesson3-m1.md

The currently live course page does seem to have the correct text though: [https://learn.codethedream.org/ctd-node-lesson4-fall2023/](https://learn.codethedream.org/ctd-node-lesson4-fall2023/)

> You continue watching [**this video**](https://youtu.be/Oe421EPjeBE?t=22246). This week, you watch from 6:10:46 of the video to the end.

Reply:
This whole part I think was accidentally copied from lesson3 in the version that's currently on staging.

To fix I just copied what was on the live site: https://learn.codethedream.org/ctd-node-lesson4-fall2023/